alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -187,6 +191,21 @@ class AlertService(
         )
     }
 
+    fun composeMonitorErrorAlert(
+        id: String,
+        monitor: Monitor,
+        alertError: AlertError
+    ): Alert {
+        val currentTime = Instant.now()
+        // TODO make Alert constructor without trigger param so that we don't have to create this dummy trigger
+        val trigger = DocumentLevelTrigger(id = "", name = "dummy-trigger", severity = "", listOf(), Script(""))
```

Comment:
we can't call this dummy trigger. maybe name could be "${monitorId}ExecutionError"

Reply:
done